packages/core/src/components/tooltip/test/disabled/tooltip.e2e.ts

```diff
@@ -0,0 +1,37 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/tooltip/test/disabled/index.html';
+
+test.describe('tds-tooltip', () => {
+  test('renders the tooltip correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('tooltip closes when cursor is removed from the button', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // Assuming your button and tooltip setup is already appropriate for this test
+    const button = page.locator('tds-button#button-2');
+
+    // Hover over the button to show the tooltip
+    await button.hover();
+    // Assuming you can uniquely identify the tooltip that should be visible on hover
+    const tooltip = page.locator('text=Text inside Tooltip');
+
+    await expect(tooltip).toBeVisible();
+    // Make sure tooltip is visible first
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0.5 });
```

Comment:
0.5 indicates 50% difference in screenshots, is the value intentionally that high?

Reply:
A mistake I think!